solidity/contracts/libraries/staking/MinimumStakeSchedule.sol

```diff
@@ -9,12 +9,20 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 library MinimumStakeSchedule {
     using SafeMath for uint256;
 
+    // Apr-28-2020 02:52:46 AM UTC when KEEP token has been deployed
+    // TX:  0xea22d72bc7de4c82798df7194734024a1f2fd57b173d0e065864ff4e9d3dc014
+    uint256 public constant scheduleStart = 1588042366;
+    
     // 2 years in seconds (seconds per day * days in a year * years)
     uint256 public constant schedule = 86400 * 365 * 2;
```

Comment:
we didn't want to use SafeMath here? `86400.mul(365).mul(2)` and 10000.mul(1e18)

Reply:
Not super worried about them given that those are constant set during deployment and we know how they are calculated, but...

it's probably a good practice to use SafeMath everywhere. Better be safe than sorry.

Reply:
Heh, no - it's not possible:
```
,/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/contracts/libraries/staking/MinimumStakeSchedule.sol:19:36: TypeError: Initial value for constant variable has to be compile-time constant.
    uint256 public constant base = uint256(10000).mul(1e18);
```